src/CMakeLists.txt

```diff
@@ -74,10 +74,16 @@ if (ENABLE_PROFILER)
     target_link_libraries(decomp2d PRIVATE caliper)
   endif()
 endif()
+
 if (HALO_DEBUG)
   target_compile_definitions(decomp2d PUBLIC -DHALO_DEBUG)
 endif()
 
+if (IO_BACKEND MATCHES "adios2")
+  add_definitions("-DADIOS2")
```

Comment:
Should be `target_compile_definitions(decomp2d PUBLIC -DADIOS2)` ?

Reply:
Fixed in https://github.com/2decomp-fft/2decomp-fft/pull/195/commits/887938756e6dfbb43077fd567c61e73a93bdf24e